app/src/main/java/com/infomaniak/mail/data/api/MailApi.kt

```diff
@@ -199,6 +196,20 @@ object MailApi {
         MailboxContentController.deleteMessages(deletableMessages)
     }
 
+    private fun fetchDraftFromApi(draftResource: String, parentUid: String): Draft? {
+        val apiDraft = ApiRepository.getDraft(draftResource).data
+        apiDraft?.let { draft ->
+            draft.apply {
+                initLocalValues(parentUid)
+                // TODO: Remove this `forEachIndexed` when we have EmbeddedObjects
+                attachments.forEachIndexed { index, attachment -> attachment.initLocalValues(index, parentUid) }
+            }
+            MailboxContentController.upsertDraft(draft)
+        }
+
+        return apiDraft
```

Comment:
use return `ApiRepository.getDraft(draftResource).data?.also`

Reply:
Done in https://github.com/Infomaniak/android-mail/pull/154